src/nvim/testdir/test_eval.in

```diff
@@ -122,7 +121,19 @@ call SetReg('/', ['abc/'])
 call SetReg('/', ["abc/\n"])
 call SetReg('=', ['"abc/"'])
 call SetReg('=', ["\"abc/\n\""])
-
+$put ='{{{1 System clipboard'
+" Save and restore system clipboard.
+" If no connection to X-Server is possible, test should succeed.
+:let _clipreg = ['+', getreg('+'), getregtype('+')]
+:let _clipopt = &cb
+:let &cb='unnamedplus'
+:1y
+:AR +
+:tabdo :windo :echo "hi"
```

Comment:
Is this addition mainly to test the `:tabdo :windo` bug? Then maybe it could be converted to a new-style test that doesn't clobber system clipboard. (If not please disregard this comment)